docs/tutorials/Soil/richards_equation.jl

```diff
@@ -148,9 +156,35 @@ t0 = FT(0)
 timeend = FT(60 * 60 * 24 * 36)
 dt = FT(100);
 
+# Now, we choose the timestepping algorithm we want to use.
```

Comment:
do we want to leave all the richards examples as using the implicit solver (though currently imp_tendency does nothing)?

might be confusing, but also might be worth it since we'll follow up with the next PR & then this will already be prepped

Reply:
I changed the `RichardsModel` examples back to use just the explicit tendency, which I think is more clear